src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```
Змінні потрібно робити якомога закртиішими щоб слідувати інкапсуляції